File: 448.cpp
class Solution {
public:
    vector<int> findDisappearedNumbers(vector<int>& nums) {
        for(int i=0;i<nums.size();++i){
            int index = abs(nums[i])-1;
            nums[index] = -abs(nums[index]);
        }
        
        vector<int> res;
        for(int i=0;i<nums.size();++i){
            if(nums[i]>0){
                res.push_back(i+1);
            }
        }
        
        return res;
    }
};

File: Python/448.py
class Solution(object):
    def findDisappearedNumbers(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        i = 0
        res = []
        while i<len(nums):
            pos = abs(nums[i])-1
            nums[pos] = -abs(nums[pos])
            i+=1
        for i, n in enumerate(nums):
            if n > 0:
                res.append(i+1)
        return res
